yt_dlp/extractor/fourchannel.py

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    try_get,
+)
+
+
+class FourChannelIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://)boards.4channel.org/(?P<board>[^/]+)/thread/(?P<thread_id>\d+)'
+    _TESTS = [{
+        # This test will break in future once the thread gets removed even from the archives.
+        'url': 'https://boards.4channel.org/wsg/thread/4088700',
+        'info_dict': {
+            'id': '4088700',
+        },
+        'playlist_mincount': 33,
+    }, {
+        # This is a stickied post so this should last forever hopefully.
+        'url': 'https://boards.4channel.org/k/thread/23385784',
+        'info_dict': {
+            'id': '1',
+        },
+        'playlist_mincount': 20000,
+    }]
```

Comment:
Since there is no videoIE, add test for `entries` too

Reply:
should be good to merge now